float64[7] tau